client/Dockerfile.packages

```diff
@@ -0,0 +1,9 @@
+FROM node:10-alpine
+
+WORKDIR /opt
```

Comment:
`/opt` usually already contains other packaged software:
```
$ docker run -it node:10-alpine sh -c 'ls -l /opt'
total 4
drwxr-xr-x    4 root     root          4096 Aug 16 02:48 yarn-v1.17.3
```
Would be `/opt/$package` to use that directory, but just `/app` would be fine?